addons/binding/org.openhab.binding.windcentrale/META-INF/MANIFEST.MF

```diff
@@ -25,5 +25,7 @@ Import-Package:
  org.eclipse.smarthome.io.net.http,
  org.openhab.binding.windcentrale,
  org.openhab.binding.windcentrale.handler,
+ org.osgi.framework,
+ org.osgi.service.component.annotations;resolution:=optional,

```

Comment:
you can remove this line again - it was a bug in the parent pom that this was needed yesterday. But it has been fixed with https://github.com/openhab/openhab-core/pull/237

Reply:
There are a lot of other bindings with the same line in their manifest. Does that mean a PR should be created to remove it from all those as well?